README.md

```diff
@@ -1,3 +1,5 @@
+Psst... Sodium for Minecraft 1.16.2 is being worked on, there's just some other architectural improvements and new optimizations in the pipeline. There's no ETA, but hold tight!  
```

Comment:
How about replacing it with 1.16.3 now?

Reply:
I just copied over the notice from the CurseForge page. I'm also not sure if Jelly is already working on a 1.16.3 release. I'll keep it at 1.16.2 for now.

Reply:
Hey, I was reading some issues and noticed that the CurseForge page has changed to mention 1.16.3 instead of 1.16.2:
> Psst... Sodium for Minecraft 1.16.3 is being worked on, there's just some other architectural improvements and new optimizations in the pipeline. There's no ETA, but hold tight!

Reply:
Modified to say "1.16.3" in my PR. (I think. I'm still getting used to GitHub's web interface after my computer crashed.)